File: GTTS/INL.TrainingService.Models/SaveTrainingEventParticipantXLSX_Result.cs
using System;
using INL.Services.Models;

namespace INL.TrainingService.Models
{
    public class SaveTrainingEventParticipantXLSX_Result : BaseResult, ISaveTrainingEventParticipantXLSX_Result
    {
        public long ParticipantXLSXID { get; set; }
        public string ParticipantStatus { get; set; }
        public string FirstMiddleName { get; set; }
        public string LastName { get; set; }
        public string NationalID { get; set; }
        public char? Gender { get; set; }
        public string IsUSCitizen { get; set; }
        public DateTime? DOB { get; set; }
        public string POBCity { get; set; }
        public string POBState { get; set; }
        public string POBCountry { get; set; }
        public int? ModifiedByAppUserID { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/save-training-event-participant-xlsx_result.ts


import { ISaveTrainingEventParticipantXLSX_Result } from './isave-training-event-participant-xlsx_result';

export class SaveTrainingEventParticipantXLSX_Result implements ISaveTrainingEventParticipantXLSX_Result {
  
	public ParticipantXLSXID: number = 0;
	public ParticipantStatus: string = "";
	public FirstMiddleName: string = "";
	public LastName: string = "";
	public NationalID: string = "";
	public Gender?: string;
	public IsUSCitizen: string = "";
	public DOB?: Date;
	public POBCity: string = "";
	public POBState: string = "";
	public POBCountry: string = "";
	public ModifiedByAppUserID?: number;
  
	public ErrorMessages: string[] = null;
}






